### ISSUE TITLE
Outdated. This branch is dead. SolarCoin is a POST see comment for link to repository

### ISSUE THREAD

**Issue by nickgogerty** [solarcoin/solarcoin-legacy-pow-wallet#13]
https://github.com/onsightit/solarcoin/

**Comment by lpninja** [comment]
agreed. We need to add the latest branches into this so the repository is under https://github.com/solarcoin even for the latest POST branch that was developed and successfully deployed by Onsightit in September 2015 and it has been running smoothly since then.
